docs/examples/create_classifier.ipynb

```diff
@@ -0,0 +1,584 @@
+       "   Feature.Cytoplasm.Haralick.DifferenceVariance.Range  \\\n",
+       "0                                           0.002288     \n",
+       "1                                           0.001967     \n",
+       "2                                           0.002020     \n",
+       "3                                           0.001623     \n",
+       "4                                           0.002390     \n",
+       "\n",
+       "   Feature.Cytoplasm.Haralick.DifferenceEntropy.Mean  \\\n",
+       "0                                           2.040481   \n",
+       "1                                           2.072380   \n",
+       "2                                           2.085567   \n",
+       "3                                           2.382495   \n",
+       "4                                           2.002746   \n",
+       "\n",
+       "   Feature.Cytoplasm.Haralick.DifferenceEntropy.Range  \\\n",
+       "0                                           0.322045    \n",
+       "1                                           0.319199    \n",
+       "2                                           0.344207    \n",
+       "3                                           0.307914    \n",
+       "4                                           0.406743    \n",
+       "\n",
+       "   Feature.Cytoplasm.Haralick.IMC1.Mean  \\\n",
+       "0                             -0.057505   \n",
+       "1                             -0.111360   \n",
+       "2                             -0.067079   \n",
+       "3                             -0.059416   \n",
+       "4                             -0.097690   \n",
+       "\n",
+       "   Feature.Cytoplasm.Haralick.IMC1.Range  \\\n",
+       "0                               0.073408   \n",
+       "1                               0.086315   \n",
+       "2                               0.091177   \n",
+       "3                               0.060096   \n",
+       "4                               0.105828   \n",
+       "\n",
+       "   Feature.Cytoplasm.Haralick.IMC2.Mean  Feature.Cytoplasm.Haralick.IMC2.Range  \n",
+       "0                              0.476290                               0.252285  \n",
+       "1                              0.658314                               0.175118  \n",
+       "2                              0.521425                               0.290303  \n",
+       "3                              0.520708                               0.198088  \n",
+       "4                              0.599820                               0.219931  \n",
+       "\n",
+       "[5 rows x 134 columns]"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Read CSV file\n",
+    "url = \"https://data.kitware.com/api/v1/file/hashsum/sha512/e8c829b60d316ff84d2ffafb5accd605eb8dcd02dec709105ec9127aa2d7969e2feca74f66394b26f0e90375cd0d1cda3d1831023449f66cf50a637906444578/download\"\n",
```

Comment:
Add a comment that this downloads a sample file, but they could substitute their own file.